### DISCUSSION TITLE
Simon

### DISCUSSION THREAD

**Discussion by Chompalot** [makerforgetech/modular-biped#49] [Introductions]
Greetings, my designation is Simon. I've always been into SciFi, for which I blame Doctor Who, Starwars and Azimov, and maybe the moon landings, which happened in my early years.
I've competed in Robotwars, both the old and new series, and alsoTechnogames. I've also built a full sized K9, which is mostly functional, but not as an independant entity. The software has always been the stumbling block.
I also ran acourse as a STEM ambassador, at Derby Museum, based on a robot developed from Digit, but a 3D printed and more polished version.
I'm hoping building my own modular biped will give me experience with ROS which will help with my other robot projects.

**Comment by WhoIsMrSentry** [comment]
Nice to meet you @Chompalot 
It's good to see new people in the community, I believe together we can bring the repository to great places.

Feel free to open a new discussion if you need help with something or a new idea.

**Comment by danic85** [comment]
Thanks for posting! I'd love to work on an implementation that uses ROS 2 as I'm definitely looking to use that more in future. Keep us updated 🤞

**Comment by LMG26** [comment]
Hello everyone my name is Luis and love to connect with people how enjoy working with robots I have been building  robots for 12 years. I would love to share some of my projects.

**Reply by WhoIsMrSentry** [reply]
Hello Luis welcome to the discussion and community, of course you can share, never hesitate to open a new topic.

**Reply by danic85** [reply]
Hi Luis, good to see you on the community! Feel free to share your projects (https://www.instagram.com/systemtechnologyworks/ for anyone interested)

**Reply by LMG26** [reply]
> Hi Luis, good to see you on the community! Feel free to share your projects (https://www.instagram.com/systemtechnologyworks/ for anyone interested)

Thank you

**Reply by Chompalot** [reply]
Hi Luis,
You have some brilliant robots :)
what are the gears you are using on your builds?
![image](https://github.com/dmt-labs/modular-biped/assets/34891324/2829ea6c-6065-49f3-a3a5-213960a8ebad)
they dont look 3D printed

**Reply by LMG26** [reply]
I use servocity.com greats.

**Comment by Zen-Balance** [comment]
Hi everyone, my name's Zen. I'm  a UX and UI designer currently working to merge further XR design and applications, robotics, and elements of Gamification together for an interesting number of special use cases. I'm looking forward to meeting any of those of you that are still active these days on this thread. Dan, I must say you've done great work and your prototype build logs are fun to watch and learn from. Please let me know if you ever receive any messages from my team reaching out to you with regards to a co-op project proposal. To the rest of you that read this I look forward to making your acquaintance. Thank you for reading this.

**Reply by WhoIsMrSentry** [reply]
Hi @Zen-Balance nice to meet you I am Emir, I am a contributor to this repository, thanks for introducing yourself and telling us about your projects. Your goal of bringing together XR design, robotics and gamification is very interesting. 

Your comment on Dan’s work is also nice, it can be really useful to learn by watching the prototyping process. If anyone from your team receives a message asking for a collaboration, we will let you know. It’s nice to meet you, hope to share more here.

I should also mention that the part about XR really caught my attention, I use Nvidia Jetson to add software that requires high processing power like this.

**Reply by danic85** [reply]
Hi @Zen-Balance , welcome to the project!

Thanks for that - it has taken a while :sweat_smile: !

Just FYI, the community is more active on our [Discord](https://bit.ly/makerforge-community), but this forum is still open (and arguably a more appropriate place than Discord for some of the questions :laughing: ), so feel free to post on either.

**Reply by Zen-Balance** [reply]
Nice to hear from you both, thank you for taking the time to respond. I'm using HoloLens 2 atm still while prototyping but looking forward to the new meta Orion, Same capabilities but much less conspicuous. 
@danic85 do you have a way I can reach out to you for a co-op project venture? Been trying to find a way to reach you but ran into a few dead ends for a bit there. A small team of mine and I would like to discuss more with you, emir, and the rest of your team. I have an interesting idea you might have fun playing around with. I even found a potential investor for down the line as well and a testing market. Been busy the last few months and am ecstatic to have made contact.

**Reply by danic85** [reply]
@Zen-Balance Discord or Instagram is probably the best bet, if either works for you?

**Comment by Zen-Balance** [comment]
Yeah, we will reach out with Instagram and go from there. :)

Yahoo Mail: Search, Organize, Conquer 
 
  On Thu, Oct 31, 2024 at 1:01 PM, Dan ***@***.***> wrote:   


@Zen-Balance Discord or Instagram is probably the best bet, if either works for you?
